notebooks/02-ingestion/14-native-filters.ipynb

```diff
@@ -0,0 +1,815 @@
+    "\n",
+    "When finished, you will see the table description."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4fb89079-7e2a-404b-be85-d9fc7c97d0f1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ioConfig = {\n",
+    "    \"type\": \"kafka\",\n",
+    "    \"consumerProperties\": { \"bootstrap.servers\": kafka_host },\n",
+    "    \"topic\": datagen_topic,\n",
+    "    \"inputFormat\": { \"type\": \"json\" },\n",
+    "    \"useEarliestOffset\": \"false\" }\n",
+    "\n",
+    "tuningConfig = { \"type\": \"kafka\" }\n",
+    "\n",
+    "table_name = datagen_topic\n",
+    "\n",
+    "dataSchema = {\n",
+    "    \"dataSource\": table_name,\n",
+    "    \"timestampSpec\": { \"column\": \"time\", \"format\": \"iso\" },\n",
+    "    \"granularitySpec\": { \"rollup\": \"false\", \"segmentGranularity\": \"hour\" },\n",
+    "    \"dimensionsSpec\": { \"useSchemaDiscovery\" : \"true\"}\n",
+    "    }\n",
+    "\n",
+    "ingestion_spec = {\n",
+    "    \"type\": \"kafka\",\n",
+    "    \"spec\": {\n",
+    "        \"ioConfig\": ioConfig,\n",
+    "        \"tuningConfig\": tuningConfig,\n",
+    "        \"dataSchema\": dataSchema\n",
+    "    }\n",
+    "}\n",
+    "\n",
+    "requests.post(f\"{druid_host}/druid/indexer/v1/supervisor\", json.dumps(ingestion_spec), headers=druid_headers)\n",
+    "sql_client.wait_until_ready(table_name, verify_load_status=False)\n",
+    "display.table(table_name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "472589e4-1026-4b3b-bb79-eedabb2b44c4",
+   "metadata": {},
+   "source": [
+    "## Filter data using an equality filter\n",
+    "\n",
+    "In this section you will use a [equality filter](https://druid.apache.org/docs/latest/querying/filters#equality-filter) to create a table that only contains records for where someone searches for a product.\n",
+    "\n",
+    "Run the following cell to get a preview of the data that we want to have in our new table."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "d32bed8e-bc49-4ba8-85d2-8a9bfa9232f8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sql=f'''\n",
+    "SELECT\n",
+    "  \"event_type\",\n",
+    "  COUNT(*) AS events\n",
+    "FROM \"{table_name}\"\n",
+    "GROUP BY 1\n",
+    "'''\n",
+    "\n",
+    "display.sql(sql)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96c81b63-777b-414f-81a9-2b9ccee214e5",
+   "metadata": {},
+   "source": [
+    "Run the next cell to create a new object that represents the [`transformSpec`](https://druid.apache.org/docs/latest/ingestion/ingestion-spec#transformspec). This will be added to the dataSchema in the ingestion specification, instructing Druid to apply a filter to the incoming data as it arrives.\n",
```

Comment:
```suggestion
    "Run the next cell to create a new object that represents [`transformSpec`](https://druid.apache.org/docs/latest/ingestion/ingestion-spec#transformspec). This will be added to the `dataSchema` in the ingestion specification, instructing Druid to apply a filter to the incoming data as it arrives.\n",
```